packages/utils/parcel-watcher-watchman-js/src/index.js

```diff
@@ -0,0 +1,8 @@
+import {createWrapper} from './wrapper';
+
+const wrapper = createWrapper();
```

Comment:
The function seems a bit unnecessary, why not just inline the `new ParcelWatcherWatchmanJS()` code?